src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -147,6 +147,13 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 			cfg.Redpanda.SeedServers = []config.SeedServer{}
 			cfg.Redpanda.SeedServers = seeds
 
+			// In bootstrapping a new cluster, use seeds-driven cluster
```

Comment:
seeds-driven == new way? Just to confirm -- it's not too obvious what the old vs. what the new way is